### DISCUSSION TITLE
How to make the "Home" in navigationview already clicked and shows the page content of "Home"when i run the mainwindow

### DISCUSSION THREAD

**Discussion by Chrlss** [lepoco/wpfui#992] [Q&A] [answered]
<ui:NavigationView Grid.Row="1" x:Name="NavMenu" PaneDisplayMode="LeftFluent">
    <ui:NavigationView.MenuItems>
        <ui:NavigationViewItem x:Name="Home" Content="Home" Tag="Home" Icon="{ui:SymbolIcon Home20}"   
                               TargetPageType="{x:Type pages:Home}" />
        <ui:NavigationViewItem Content="Protection" Tag="Protection" Icon="{ui:SymbolIcon Shield20}"
                               TargetPageType="{x:Type pages:Protection}" />
        <ui:NavigationViewItem Content="Security" Tag="Security" Icon="{ui:SymbolIcon LockClosed20}" 
                               TargetPageType="{x:Type pages:Security}" />
        <ui:NavigationViewItem Content="Monitor" Tag="Monitor" Icon="{ui:SymbolIcon DataHistogram20}"
                               TargetPageType="{x:Type pages:Monitor}" />
    </ui:NavigationView.MenuItems>
    <ui:NavigationView.FooterMenuItems>
        <ui:NavigationViewItem Content="Help" Tag="Help" Icon="{ui:SymbolIcon QuestionCircle20}" 
                               TargetPageType="{x:Type pages:Help}" />
        <ui:NavigationViewItem Content="Settings" Tag="Settings" Icon="{ui:SymbolIcon Settings20}"
                               TargetPageType="{x:Type pages:Settings}" />
    </ui:NavigationView.FooterMenuItems>
    <ui:NavigationView.ContentOverlay>
        <Grid>
            <ui:SnackbarPresenter x:Name="SnackbarPresenter" Margin="10,0,-10,0" />
        </Grid>
    </ui:NavigationView.ContentOverlay>
</ui:NavigationView>

**Comment by pomianowski** [comment, marked as answer]
Hey, you can add a Loaded event to your MainWindow. Once the window is loaded, you can tell the navigation to navigate to a specific page.
https://github.com/lepoco/wpfui/blob/baaf749864df48be57c18821c9dcd47a2b5faa73/src/Wpf.Ui.Gallery/Services/ApplicationHostService.cs#L60-L68

https://github.com/lepoco/wpfui/blob/baaf749864df48be57c18821c9dcd47a2b5faa73/src/Wpf.Ui.Demo.Simple/MainWindow.xaml.cs#L13-L25

**Reply by Chrlss** [reply]
Thankyouuu it works!
